tests/e2e/e2e_test.go

```diff
@@ -288,7 +289,7 @@ var _ = Describe("odoe2e", func() {
 				Expect(cmpList).To(ContainSubstring(curProj))
 			})
 
-			It("should create a named component", func() {
+			It("should create a named component, add a .odoignore file and then push the contents from the source excluding the contents in the .odoignore file", func() {
```

Comment:
Too lengthy. Why you are not adding the description as i suggested one https://github.com/redhat-developer/odo/pull/1332#discussion_r259273409 or similar